bbuhot_client/src/app/bet-page/game-list-service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { ApiMethods } from '../api/api-methods';
+import { Game } from '../proto/bbuhot/service/game_pb';
+
+export class GameAccessor {
```

Comment:
It doesn't make sense. The generated `toObject` method transforms all nested message into plain `AsObject` type.

Reply:
This can give us more flexibility when accessing the data object. Usually we call this DTO (data transfer object).

I didn't do the cast from Game to Game.AsObject, in case we need to provide the game back to our backend.